Formula/ioke.rb

```diff
@@ -17,8 +17,8 @@ def install
       s.change_make_var! "IOKE_HOME", libexec
     end
 
-    bin.install_symlink libexec/"bin/ioke",
-                        libexec/"bin/ispec",
+    (bin/"ioke").write_env_script libexec/"bin/ioke", Language::Java.overridable_java_home_env
+    bin.install_symlink libexec/"bin/ispec",
                         libexec/"bin/dokgen"
```

Comment:
`ispec` and `dokgen` use shebang `#!/usr/bin/env ioke`

So, I think only `ioke` needs JAVA_HOME set.